clients/go/README.md

```diff
@@ -1,6 +1,25 @@
 # Zeebe Go Client
 
-## Development
+> [!WARNING]
+> The Zeebe Go Client will be officially deprecated with 8.6 release as part of our efforts to streamline the Camunda 8 API experience. This client will not get released with Camunda 8.6, thus no longer receive new features and will be transitioned to a community-maintained status.
```

Comment:
Suggestion to align with the announcement:

The Zeebe Go Client will be officially deprecated with the 8.6 release as part of our efforts to streamline the Camunda 8 API experience. This client will not get released starting with Camunda 8.6, will no longer receive new features, and will be transitioned to a community-maintained status.

Reply:
Also, do we need to mention zbctl here as well? The announcement does, e.g.:

The Zeebe Go Client and zbctl will be officially deprecated with the 8.6 release as part of our efforts to streamline the Camunda 8 API experience. This client and CLI utility will not get released starting with Camunda 8.6, will no longer receive new features, and will be transitioned to a community-maintained status.

Reply:
Did this get incorporated into a PR @aleksander-dytko @mesellings?

Reply:
> Did this get incorporated into a PR @aleksander-dytko @mesellings?

@akeller the actual announcement was merged for the alpha 5 release as part of the PR: https://github.com/camunda/camunda-docs/pull/4264, but I'm not sure about this change to the readme file @aleksander-dytko?